spec.html

```diff
@@ -826,8 +826,10 @@ <h1>Implicit Completion Values</h1>
         </emu-alg>
         <p>A &ldquo;return&rdquo; statement without a value in an algorithm step means the same thing as:</p>
```

Comment:
I only see 2 instances of `Return.` in the spec. I'd rather eliminate this rule and replace them with `Return NormalCompletion(~unused~).`

Reply:
Sure, done.